var/spack/repos/builtin/packages/python/tkinter.patch

```diff
@@ -0,0 +1,17 @@
+diff -Naur a/setup.py b/setup.py
+--- a/setup.py	2019-01-13 18:59:14.000000000 -0600
++++ b/setup.py	2019-01-13 19:00:31.000000000 -0600
+@@ -1787,13 +1787,6 @@
+         if self.detect_tkinter_explicitly():
+             return
+ 
+-        # Rather than complicate the code below, detecting and building
+-        # AquaTk is a separate method. Only one Tkinter will be built on
+-        # Darwin - either AquaTk, if it is found, or X11 based Tk.
+-        if (host_platform == 'darwin' and
+-            self.detect_tkinter_darwin(inc_dirs, lib_dirs)):
```

Comment:
`detect_tkinter_darwin` ignores `inc_dirs` and `lib_dirs` and instead searches the system locations for Tk/Tcl. This results in Spack's `tk@8.6` being linked in some places and the system `tk@8.5` being linked in others. Although the modules builds, when you try to use it, it complains about these version mismatches.